test/test_fourier.jl

```diff
@@ -0,0 +1,407 @@
+function compare_tables(table1, table2; rtol=0.001, discard = [])
+
+    s_discard = Symbol.(discard)
+
+    for key in propertynames(table1)
+        if key in s_discard
+            continue
+        end
+        oe, oc = getproperty(table1,key), getproperty(table1,key)
+        if oe isa Integer || oe isa String
+            @assert oe==oc
```

Comment:
[`@assert`](https://docs.julialang.org/en/v1/base/base/#Base.@assert) is supposed to be used only for debugging purposes.  Please read the docstrings which strongly warns against any other uses.

Reply:
I thought of providing only one tests for each compare_tables execution, so asserted the conditions and finally tested for true, if I test for every key value, there are around 500 tests related to them.